app/Repositories/ChatRepository.php

```diff
@@ -313,9 +313,9 @@ public function systemMessage($message, $bot = null)
         if ($bot) {
             $this->message($systemUserId, $this->systemChatroom(), $message, null, $bot);
         } else {
-            $systemBotId = Bot::where('slug', 'systembot')->first()->id;
+            $systemBotId = Bot::where('command', 'systembot')->first()->id;
 
-            $this->message($systemUserId, $this->systemChatroom(), $message, null, $systemBotId);
+            $this->message($systemBotId, $this->systemChatroom(), $message, null, $systemBotId);
```

Comment:
It looks like this change breaks several tests.

The method signature is `public function message($user_id, $room_id, $message, $receiver = null, $bot = null)`, and this change passes a Bot ID instead of a User ID.

It's entirely possible that https://github.com/HDInnovations/UNIT3D-Community-Edition/commit/aa32a893fa7767f91b4dcefcf76016179b6e8066 introduced a regression, in which case we should take a look and update the tests to describe the correct behavior if necessary.

Did you notice a specific regression due to the aforementioned commit @shlandturtle ?

Reply:
That was unneeded change which I did due to not realizing where the issues was. After the commit you linked all chat announces stopped working, but it was due to the "'slug', 'systembot'", not this edit. Removed ChatRepository changes from this commit now. My bad.